test/deploy-pipeline-lite.sh

```diff
@@ -95,6 +98,11 @@ do
   kubectl rollout status $deployment -n ${NAMESPACE}
 done
 
+for statefulset in $(kubectl get statefulset -n ${NAMESPACE} -o name)
```

Comment:
Can you change above comment to `after we introduce daemonsets, we need to wait...`